data/hai/hai reveal 1 intro.txt

```diff
@@ -670,16 +692,13 @@ mission "Hai Leaks Response 2C"
 			label met
 			action
 				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai.`
-			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. "You do remember me, right? I don't want this to be an awkward situation where only I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis.`
+			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. He gestures to the woman behind him and says, "Let me introduce you to Terry."`
 			label next
-			`	"Now, allow me to introduce you to Terry," Remington says while gesturing to the woman behind him.`
```

Comment:
By moving this line above the `next` label, the part of the branch where you haven't met Remington before now doesn't have Remington introducing Terry. So either this line shouldn't be merged with the one before it, or you should duplicate this line to go above the `goto next` line so that both branches properly introduce Terry.

Reply:
I reworked this area. If you don't know Remington, you get the full text. If you know Remington, then you get to skip a bit. With the other changes to the scene's dialog, the text wall isn't so bad now.